manifests/portstree.pp

```diff
@@ -3,13 +3,14 @@
 # parameter
 
 define poudriere::portstree (
-  Enum['present', 'absent'] $ensure           = 'present',
-  String[1]                 $portstree        = $name,
-  Optional[String[1]]       $branch           = undef,
-  Poudriere::Fetch_method   $fetch_method     = 'svn',
-  Boolean                   $cron_enable      = false,
-  Boolean                   $cron_always_mail = false,
-  Poudriere::Cron_interval  $cron_interval    = { minute => 0, hour => 22, monthday => '*', month => '*', week => '*' },
+  Enum['present', 'absent']      $ensure           = 'present',
+  String[1]                      $portstree        = $name,
+  Optional[String[1]]            $branch           = undef,
+  Poudriere::Fetch_method        $fetch_method     = 'svn',
+  Optional[Stdlib::Absolutepath] $mountpoint       = undef,
```

Comment:
Since this module is not using puppet-strings, the new parameter should be at least documented in the README.md

Reply:
Hum, yes, I was surprised to not see (at)params comments…  What would it cost to use this?  I mean, may I just add these comments and we are done or do we have to tweak nasty things to finish the work?

Reply:
I see non-working options :astonished: (e.g. setting `poudriere::env::makefile` leads to a file with both `source` and `content`)…  I'll add an item in the #52 issue for adding tests which combine all options.

Reply:
Heh, I just meant add the new param to the README, adding puppet-strings for everything is awesome!